docs/release-notes.md

```diff
@@ -1,115 +1,211 @@
 
-**Hostname Override:** In response to customer feedback, we have added the ability to override the hostname using a -h command in RPC-Go.  Instead of sending up the HostOS name, RPC will now send up the host name specified.  This is particularly helpful when running RPC from a container.
+**Device Migration:** Through working with our customers, we've had several requests for a way to move an Intel&reg; AMT device from one management console to the the Open AMT Cloud Toolkit.  With 2.2, you can perform this migration with RPC by using the activate command and provide the profile you wish to use to activate the device.  By providing the AMT credential at the prompt, RPS will use this credential to add the device to Open AMT Cloud Toolkit and then apply the configuration profile, completing the migration without needing to deactivate the device.
 
-**CIRA Connection & Disconnection Events:** When devices connect and disconnect from the MPS, an event will now be sent to the MQTT broker.  
+**Healthcheck API:** This new API provides you the ability to check the status of MPS and RPS to know that they have a connection with both the database and secret provider.  This is particularly helpful when Open AMT Cloud Toolkit is deployed in Kubernetes or Docker Swarm so that you know they are up and ready to respond to requests.
 
-**Branch Renaming:** We have renamed "master" to "main" across all of our repositories.  To learn more about why we made this change, please see the [Diversity at Intel](https://www.intel.com/content/www/us/en/diversity/diversity-at-intel.html) web site.
+**RPC -json flag:** We have added a new -json flag to the RPC application.  This will convert the output to json format making the output machine parsable.  Big thanks to [Portainer](https://www.portainer.io/) for this contribution!
```

Comment:
**RPC -json flag:** The rpc-go application sports a new -json flag. With this flag, convert application output to json format making it machine parsable.  Big thanks to [Portainer](https://www.portainer.io/) for this contribution!